config/read.go

```diff
@@ -748,6 +748,11 @@ func Read(logger *util.Logger, filename string) (Config, error) {
 
 		sections := configFile.Sections()
 		for _, section := range sections {
+			sectionName := section.Name()
+			if sectionName == "pganalyze" || sectionName == "DEFAULT" {
```

Comment:
```suggestion
			if sectionName == "pganalyze" || sectionName == ini.DefaultSection {
```

Is default section really handled already? In which part it's handled?

Reply:
Ah, good catch that a constant exists for this. It's handled in the sense that we don't use it. It will go through this loop but not really do anything. I think it's clearer to skip it explicitly. I'll update the comment to reflect that more clearly.